clerk/verification.go

```diff
@@ -45,7 +45,19 @@ func (s *VerificationService) useClientActiveSession(token string) (*Session, er
 		return nil, errors.New("no active sessions for given client")
 	}
 
-	return s.client.Sessions().Read(*clientResponse.LastActiveSessionID)
+	var lastActiveSession *Session
+	for _, session := range clientResponse.Sessions {
+		if session.ID == *clientResponse.LastActiveSessionID {
+			lastActiveSession = session
+			break
+		}
+	}
+
+	if lastActiveSession == nil {
+		return nil, errors.New("active session not included in client's sessions")
+	}
+
+	return lastActiveSession, nil
```

Comment:
This can be simplified as below without the need of new variable initialization.

```
for _, session := range clientResponse.Sessions {
		if session.ID == *clientResponse.LastActiveSessionID {
                         return session, nil
		}
}

return nil, errors.New("active session not included in client's sessions")
```